### ISSUE TITLE
Cheaten ist theoretisch möglich - man kann die Karten aufdecken und dann wieder zudecken

### ISSUE THREAD

**Issue by dev-hias** [andreasteich/ramory#2]
Man kann die Memory-Karten aufdecken und dann wieder zudecken, ohne dass der Counter mitzählt, daher könnte man theoretisch alle Karten nacheinander aufdecken und dann jeweils die richtigen Paare zusammensuchen

**Comment by andreasteich** [comment]
Ja, das ist ein sehr guter Punkt, das müssen wir gleich unterbinden.

**Comment by andreasteich** [comment]
Bitte erneut prüfen, womöglich schon outdated! 😃
